leakcanary-android/src/main/java/com/squareup/leakcanary/AndroidExcludedRefs.java

```diff
@@ -305,6 +306,14 @@
 
   // ######## Manufacturer specific Excluded refs ########
 
+  INSTRUMENTATION_RECOMMEND_ACTIVITY(MEIZU.equals(MANUFACTURER) && SDK_INT >= LOLLIPOP) {
```

Comment:
Should not be `SDK_INT >= LOLLIPOP`. Only target the versions you know for sure are impacted.